samples/meetings-attendance-report/nodejs/.env

```diff
@@ -0,0 +1,4 @@
+MicrosoftAppId=<<Microsoft-App-Id>>
+MicrosoftAppPassword=<<Microsoft-App-Password>>
+MicrosoftAppTenantId=<<Microsoft-App-Tenant-Id>>
+UserId=<<Granting-Policy-User-Id>>
```

Comment:
How are we going to get this id and add here ?
Please add the steps in Readme file.

Reply:
It is mentioned.